themes/gruvbox-dark-medium.json

```diff
@@ -1,6 +1,62 @@
 {
   "name":"Gruvbox Dark Medium",
-  "settings":[
+  "type": "dark",
+  "colors" : {
+        "activityBar.background": "#282828",
+        "activityBar.foreground": "#ebdbb2",
+        "activityBarBadge.background": "#458588",
+        "activityBarBadge.foreground": "#ebdbb2",
+        "sideBar.background": "#282828",
```

Comment:
Please make these indented to 4 spaces instead (they're at 8 atm whist the upper level is at 2 only). 
Or you can make the whole file having indent size of `4` and run a `Format Document` command in Code. 
Either way are fine, but they should be consistent, not mixed.